dotcom-rendering/src/components/Card/components/TrailTextWrapper.tsx

```diff
@@ -18,9 +18,9 @@ type Props = {
 	imageType?: CardImageType | undefined;
 	/** By default, trail text is hidden at specific breakpoints. This prop allows consumers to show trails across all breakpoints if set to false */
 	shouldHide?: boolean;
-	/** If the card is a flexible splash card, the trail text will be styled differently */
-	isFlexSplash?: boolean;
 	trailTextSize?: TrailTextSize;
+	/** Optionally overrides the trail text colour. Defaults to headline colour if not provided */
```

Comment:
Looking at line 78, looks like the default is `palette('--card-trail-text')` - so should this say something like "Uses the default trail text colour if not provided"